flamedisx/inference.py

```diff
@@ -263,8 +265,10 @@ def _minimize(self):
             # Of all scipy-optimize methods, only trust-constr takes
             # both a Hessian and bounds argument.
             kwargs.setdefault('method', 'trust-constr')
+            print('Using scipy trust-constr')
```

Comment:
If you've checked it's passing the argument correctly remove the print (also on line 271)